src/Controllers/BatchProcessorController.js

```diff
@@ -373,6 +374,24 @@ var StreamStats;
                 }).finally(function () {
                 });
             };
+            BatchProcessorController.prototype.deleteBatch = function (batchID, deleteCode, batchStatusEmail) {
+                var _this = this;
+                var text = "Are you sure you want to delete Batch ID " + batchID + "?";
+                if (confirm(text) == true) {
+                    var url = configuration.baseurls['BatchProcessorServices'] + configuration.queryparams['SSBatchProcessorDeleteBatch'].format(deleteCode);
```

Comment:
The 'SSBatchProcessorDeleteBatch' end point needs to be added to the 'configuration.queryparams' in 'appConfig.js'

Reply:
Woops, sorry! Added it